samples/AnomalyDetector/src/Generated/Docs/AnomalyDetectorClient.xml

```diff
@@ -2,78 +2,102 @@
 <doc>
   <members>
     <member name="DetectUnivariateEntireSeriesAsync(UnivariateDetectionOptions,CancellationToken)">
-<example>
-This sample shows how to call DetectUnivariateEntireSeriesAsync with required parameters.
+      <example>
+This sample shows how to call DetectUnivariateEntireSeriesAsync.
 <code><![CDATA[
-var credential = new AzureKeyCredential("<key>");
-var endpoint = new Uri("<https://my-service.azure.com>");
-var client = new AnomalyDetectorClient(endpoint, credential);
+Uri endpoint = new Uri("<https://my-service.azure.com>");
+AzureKeyCredential credential = new AzureKeyCredential("<key>");
+AnomalyDetectorClient client = new AnomalyDetectorClient(endpoint, credential);
 
-var options = new UnivariateDetectionOptions(new TimeSeriesPoint[] 
+UnivariateDetectionOptions options = new UnivariateDetectionOptions(new List<TimeSeriesPoint>()
 {
-    new TimeSeriesPoint(3.14f)
+new TimeSeriesPoint(123.45F)
+});
+Response<UnivariateEntireDetectionResult> response = await client.DetectUnivariateEntireSeriesAsync(options);
+]]></code>
+This sample shows how to call DetectUnivariateEntireSeriesAsync with all parameters.
+<code><![CDATA[
+Uri endpoint = new Uri("<https://my-service.azure.com>");
+AzureKeyCredential credential = new AzureKeyCredential("<key>");
+AnomalyDetectorClient client = new AnomalyDetectorClient(endpoint, credential);
+
+UnivariateDetectionOptions options = new UnivariateDetectionOptions(new List<TimeSeriesPoint>()
 {
-        Timestamp = DateTimeOffset.UtcNow,
-    }
+new TimeSeriesPoint(123.45F)
+{
+Timestamp = DateTimeOffset.Parse("2022-05-10T14:57:31.2311892-04:00"),
```

Comment:
Can we format this sample code? It is not easy to read.

Reply:
Of course the sample code is formatted. You see the bad format here because of a bug in Rolsyn: https://github.com/dotnet/roslyn/issues/8269 (which I leave TODO items in our code as well [here](https://github.com/Azure/autorest.csharp/blob/d92e59263cca015f9a250c95c3a3715d17e67fc4/src/AutoRest.CSharp/LowLevel/Extensions/CodeWriterExampleExtensions.cs#L235))

We could do a manual fix - we are already doing some in `XmlFormatter` class if you notice.
It is complicated to do that, and it only happens in collection initializations, therefore I would like to leave it to another PR

Reply:
When will Rolsyn bug will be resolved? Just a little concern about the format, if we generate the sample in that format, customer may complain it.

Reply:
I do not know therefore I created the item above.
This issue is not introduced by this refactor which is reason I have no intention to fix it here.

Reply:
it is simpler to fix than I thought - I have fixed this issue in this PR.